classlib/src/main/java/org/teavm/classlib/java/util/JSArraysGenerator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2025 ihromant.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.classlib.java.util;
+
+import org.teavm.backend.javascript.spi.Injector;
+import org.teavm.backend.javascript.spi.InjectorContext;
+import org.teavm.model.MethodReference;
+
+public class JSArraysGenerator implements Injector {
```

Comment:
I don't think that using generators here is an optimal approach. This may be trivial for `sort` and fill, but `sort` with comparison function and `copyOfRange` would be more problematic.

Reply:
Just use JSO